JavaScript/Task5/ArrayHelpers.js

```diff
@@ -0,0 +1,25 @@
+//http://www.codewars.com/kata/array-helpers
+
+Array.prototype.square = function() {
+  return this.map(number => number ** 2);
+};
+
+Array.prototype.cube = function() {
+  return this.map(number => number ** 3);
+};
+
+Array.prototype.average = function() {
+  return this.sum() / this.length;
```

Comment:
This function throws an error if I call this method over an empty array ie length 0. So add edge cases also.